extensions/stac-eo-spec.md

```diff
@@ -82,6 +82,64 @@ numbers of several popular instruments.
+| Landsat 8 Collection L1TP, L1GT, L1GS    |                   | True                  | True                     | False                 |  
+| Sentinel 2 2A                            |                   | False                 | False                    | True                  |  
+| Sentinel 2 1C                            |                   | False                 | True                     | False                 |  
+| Sentinel 2 1B                            |                   | True                  | False                    | False                 |  
+| Planetscope 1B                           |                   | True                  | False                    | False                 |  
+| Planetscope 3B                           |                   | True/False            | False                    | True/False            |  
+| CBERS-4 L2, L4                           |                   | True                  | False                    | False                 |
+
+##### References
+
+[Landsat Collection Definition Document](https://landsat.usgs.gov/sites/default/files/documents/LSDS-1656_Landsat_Level-1_Product_Collection_Definition.pdf)
+
+[Sentinel 2 Processing Levels](https://sentinel.esa.int/web/sentinel/user-guides/sentinel-2-msi/processing-levels)
+
+[Planet Imagery Product Specifications](https://assets.planet.com/docs/Planet_Combined_Imagery_Product_Specs_letter_screen.pdf)
+
+#### Geometric processing features
+
+| Field Name                | Type                     | Description                                                                                                                                                                                                                                                |
+| ------------------------- | ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| eo:geo_systematic         | boolean                  | Geometric location obtained from satellite's auxiliary telemetry.                                                                                                                                                     |
+| eo:geo_ortho              | boolean                  | Orthorectified.                                                                                                                                                                                                       |
+| eo:geo_gcp                | boolean                  | Registered with ground control points.                                                                                                                                                                                |
```

Comment:
There is also the possibility that the image is not projected at all. The same observations made above for the relative calibration feature apply here.